scripts/create_jekyll_ninjabuild.py

```diff
@@ -72,8 +72,12 @@ def scan_adoc(adoc_filename, apparent_filename):
         targets = []
         for page, title in sorted(category_pages):
             dest = os.path.join('$out_dir', page)
-            ninja.build(dest, 'create_categories_page', variables={'title': title})
+            ninja.build(dest, 'create_categories_page', variables={'title': title, 'layout': 'boxes'})
             targets.append(dest)
+        # 404 page
+        dest = os.path.join('$out_dir', '404.adoc')
+        ninja.build(dest, 'create_categories_page', variables={'title': 'Raspberry Pi Documentation', 'layout': 'boxes404'})
```

Comment:
This is not specific to this PR but it has tripped me up before: is there an explanation for the various parts of the toolchain and their responsibility? e.g.

1. Make
2. ASCIIDoctor
3. Jekyll
4. Ninja

In a typical Jekyll site, I would add a new file to the source directory and it would automatically get copied into the final, built version but here it seems like we have to use Ninja to opt-in to new templates and assets?

Reply:
There _will_ be, once I've finished ironing out the kinks in the build system :stuck_out_tongue_winking_eye: 
At the moment it's a bit of a weird hybrid between Make and Ninja, which sometimes causes problems with the dependency-info, so I'm aiming to move as much of it as possible solely into the Ninja world. And then I'll update the info in README.md.